allennlp/modules/transformer/attention_module.py

```diff
@@ -487,8 +482,6 @@ def __init__(
             relative_attention_num_buckets=relative_attention_num_buckets,
         )
 
-        self.attn = Attention.by_name(self.scoring_func)(scaling_factor=1, normalize=False)
```

Comment:
@AkshitaB, this is where the scaling factor is forced to `1`.

Reply:
This is for `T5Attention`.